CHANGELOG.md

```diff
@@ -46,6 +48,10 @@
       * UnionPay cards can now be processed as regular cards (through the BraintreeCard module) due to their partnership with Discover
   * BraintreeCore
     * Remove `BTConfiguration+Extensions.isUnionPayEnabled` property
+  * BraintreeSEPADirectDebit
+    * The `tokenize` method no longer takes in a `context` parameter
+    * Merchants no longer need to conform to the `ASWebAuthenticationPresentationContextProviding` protocol
+      * This is now handled by the SDK in the `BTSEPADirectDebitClient`
```

Comment:
Take it or leave it comment - I don't think this is needed for the CHANGELOG
```suggestion
```

Reply:
Yeah we added this because we weren't sure if it was clear that the responsibility of this shifted and there isn't a change in behavior. If we don't feel it's helpful we can remove it!